src/pages/AuthPage/AuthPage.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { EmailAuth } from '../../auth/EmailAuth/EmailAuth';
+import { GoogleAuth } from '../../auth/GoogleAuth/GoogleAuth';
+import { AppleAuth } from '../../auth/AppleAuth/AppleAuth';
```

Comment:
Can you make index.ts & try it on this way?

import {EmailAuth, GoogleAuth, AppleAuth} from '../../auth'